consensus/polybft/fsm_test.go

```diff
@@ -971,117 +971,6 @@ func TestFSM_VerifyStateTransactions_EndOfEpochMissingDistributeVaultFundsTx(t *
 	)
 }
 
-func TestFSM_VerifyStateTransactions_EndOfEpochMissingCommitEpochTxWhenFundRewardWallet(t *testing.T) {
```

Comment:
Did you add tests to check whether system transactions executed at the end of an epoch are in the right order?